dotnet-logging/LoggingAPI/LoggingAPI/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace LoggingAPI
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            ILoggerFactory loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddConsole();
+                builder.AddDebug();
+            });
+
+            ILogger<Student> logger = loggerFactory.CreateLogger<Student>();
+            Student student = new Student("Raghu", "IT", logger);
+
+            Department department = new Department("IT", "Information Technology", loggerFactory);
```

Comment:
You should use var whenever possible, with the exception of the ILogger variable since we want the readers to be clear about what the type of the created instance is.

Reply:
Updated to use var